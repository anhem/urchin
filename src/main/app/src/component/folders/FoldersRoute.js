import React, {Component} from 'react'
import {Route, Switch} from 'react-router-dom'
import FoldersContainer from './FoldersContainer';
import EditFolderContainer from './folder/EditFolderContainer';
import NewFolderContainer from "./folder/NewFolderContainer";

class FoldersRoute extends Component {

    render() {
        return (
            <Switch>
                <Route exact path="/folders" component={FoldersContainer}/>
                <Route path="/folders/new" component={NewFolderContainer}/>
                <Route path="/folders/:id" component={EditFolderContainer}/>
            </Switch>
        )
    }
}

export default FoldersRoute