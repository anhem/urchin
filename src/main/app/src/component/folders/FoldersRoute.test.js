import React from 'react';
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json';
import FoldersRoute from "./FoldersRoute";

describe('FoldersRoute', () => {

    it('match snapshot', () => {
        expect(toJson(shallow(<FoldersRoute/>))).toMatchSnapshot();
    });
});
