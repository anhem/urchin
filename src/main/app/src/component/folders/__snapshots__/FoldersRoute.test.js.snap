// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FoldersRoute match snapshot 1`] = `
<Switch>
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(FoldersContainer)",
        "type": [Function],
      }
    }
    exact={true}
    path="/folders"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(ConfirmNewFolderContainer)",
        "type": [Function],
      }
    }
    path="/folders/new/confirm"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(NewFolderContainer)",
        "type": [Function],
      }
    }
    path="/folders/new"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(EditFolderContainer)",
        "type": [Function],
      }
    }
    path="/folders/:id"
  />
</Switch>
`;
